GCTC-NTGC/gc-digital-talent
# Create snyk.yml

Resolves #2691
Scope of this yml file is focused on frontend dependencies 
This snyk test will test for new dependencies that are being added in PRs 
Tested and works fine.
Todo: 

- [x] - Remove Continue on Error flag to make it fail 
- [x] - Look into upload the failure report into code scanning in github 


How to test npm vulnerability scan: 

1.  checkout the branch
2. install pdfimage package locally that is vulnerable (https://snyk.io/advisor/npm-package/pdf-image)
3. Now you can test github action either locally via https://github.com/nektos/act Or push to the branch and test ( but please make sure to revert the commit ) 
4. Github action for snyk should fail showing the package as vulnerable
